src/main/java/io/strimzi/test/container/StrimziKafkaCluster.java

```diff
@@ -228,10 +229,24 @@ public StrimziKafkaCluster build() {
 
     /**
      * Get collection of Strimzi kafka containers
+     *
+     * @deprecated use {@link #getNodes()} instead.
      * @return collection of Strimzi kafka containers
      */
+    @Deprecated
     public Collection<KafkaContainer> getBrokers() {
-        return this.brokers;
+        return this.nodes;
+    }
+
+    /**
+     * Returns the underlying GenericContainer instances for Kafka brokers.
```

Comment:
That also returns controller containers, right?

Reply:
Yeah, I would rephrase it a bit.